teamcity-rest-client-api/src/main/kotlin/org/jetbrains/teamcity/rest/api.kt

```diff
@@ -190,6 +190,7 @@ interface TestRunsLocator {
     fun forTest(testId: TestId): TestRunsLocator
     fun forProject(projectId: ProjectId): TestRunsLocator
     fun withStatus(testStatus: TestStatus): TestRunsLocator
+    fun withoutDetailsField(): TestRunsLocator
```

Comment:
I'd say withoutDetails() would be a little bit better, we do not really use the word 'fields' in this client

Reply:
I thought about it too. It seems that we did not add such features to the locators at all (But maybe I'm wrong). And because this is a locator, I think it's important to mention that this is a field and not something that will affect the filtering of objects.

Reply:
Maybe we need to come up with another appropriate name?

Reply:
You are right, the culprit is that it is not really a locator, it's a separate thing called fields. 

Actually, In the similar places, inclusion of the non-required fields is handled in the `*Impl` objects via `fetchFullBean()` and `nullable/notNull` functions. In this case we may need something like that, but it involves a larger refactoring of `TestOccurrenceImpl`

Reply:
But maybe not delete. I also think that it is useful to keep the ability to fetch details for all tests by one request.

Reply:
Sorry, I'm a bit blind today, you've done that indeed, my bad